pkgs/development/libraries/quarto/default.nix

```diff
@@ -0,0 +1,55 @@
+{ callPackage
+, lib
+, pandoc
+, esbuild
+, deno
+, stdenv
+, fetchurl
+, nixos
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "quarto";
+  version = "1.0.38";
+
+  src = fetchurl {
+    url = "https://github.com/quarto-dev/quarto-cli/releases/download/v1.0.38/quarto-1.0.38-linux-amd64.tar.gz";
```

Comment:
Use the `version` variable instead of duplicating the version string.

Reply:
Solved in https://github.com/NixOS/nixpkgs/pull/186697/commits/0f77f3856fd35c19be61dbcf3f9701fc0fbe1074